gen/drupal_elixir

```diff
@@ -67,9 +66,9 @@ my $fileMemberships = $DIRECTORY . "memberships.csv";
 my $agent = perunServicesInit->getAgent;
 my $groupAgent = $agent->getGroupsAgent;
 
-foreach my $resourceData ($data->getChildElements) {
-	foreach my $groupData (($resourceData->getChildElements)[0]->getChildElements) {
-		my $groupMembersLogins = processGroups $groupData;
+foreach my $resourceId ( $data->getResourceIds() ) {
```

Comment:
Because entities are iterated over in ordered way now, user entries in `users-duplicities.csv` and `users.csv` has changed. It's because drupal (end service) considers same email as the same user. Since `groups.csv` and `memberships.csv` hasn't changed, I think it shouldn't affect the users -> they should have access to the same resources in the drupal. What do you think @stavamichal ?

Reply:
I discussed this with service owner and this change is OK for them, since they then manually adjust users in drupal instance.